sample-resource-plugin/src/main/java/org/opensearch/sample/resource/actions/rest/revoke/RevokeResourceAccessRestAction.java

```diff
@@ -72,7 +71,7 @@ private SharedWithActionGroup.ActionGroupRecipients parseRevokedEntities(Map<Str
             throw new IllegalArgumentException("entities_to_revoke is required and cannot be empty");
         }
 
-        Map<Recipient, Set<String>> entitiesToRevoke = source.entrySet()
+        Map<Recipient, Collection<String>> entitiesToRevoke = source.entrySet()
```

Comment:
why this change?

Reply:
I reverted this back to Set to ensure uniqueness.